sigma-api/src/main/scala/special/sigma/SigmaDsl.scala

```diff
@@ -200,6 +199,7 @@ trait BigInt {
 @scalan.Liftable
 @WithMethodCallRecognizers
 trait GroupElement {
+  /** Checks if this is the point at infinity. */
```

Comment:
Actually, for a group there is a notion of identity element, not a "point at infinity" (the latter is from elliptic curves terminology). Thus it would be better not to continue to mix terms, rename `isInfinity` to `isIdentity` and `negate` to `inverse`

Reply:
Created a new [issue](https://github.com/ScorexFoundation/sigmastate-interpreter/issues/865) so this can be done on top of the latest `develop` branch.
Note, the comment about `negate`.

Reply:
Why not to remove isInfinity in this PR? The problem is that a group does not have any "point at infinity", and now on top of name which is not proper you're doing comment which does not make any sense

Reply:
Why it is named isInfinity in BC then? 
Anyway, renamed the method and removed ScalaDoc. 
After you critique, I'm reluctant to write anything there.

Reply:
In an elliptic curve implementation you mean ? 

Well, an elliptic curve has a notion of infinity point, as well as addition, negation etc. A group does not have a notion of infinity point. A group has an identity element (which is instantiated with an infinity point when you instantiate a group on top of EF). Check group theory materials, e.g. https://crypto.stanford.edu/pbc/notes/group/group.html . You can make a ScalaDoc based on that.

Reply:
Please write the ScalaDoc is you think make sense. I will go ahead with other tasks.

Reply:
` /** Checks if the provided element is an identity element. */` simply

Reply:
fixed, thanks